rsocket-transport-netty/src/test/java/io/rsocket/transport/netty/WebsocketPongServer.java

```diff
@@ -16,14 +16,17 @@
 package io.rsocket.transport.netty;
 
 import io.rsocket.RSocketFactory;
+import io.rsocket.stat.RSocketStats;
+import io.rsocket.stat.RSocketStatsFactory;
 import io.rsocket.test.PingHandler;
 import io.rsocket.transport.netty.server.WebsocketServerTransport;
 
 public final class WebsocketPongServer {
 
     public static void main(String... args) throws Exception {
+        RSocketStats stats = RSocketStatsFactory.create();
         RSocketFactory
-            .receive()
+            .receive(stats)
```

Comment:
I think a good sign of this change being seamless, is if you don't need to change these examples which don't care about stats.